### PR TITLE
Why diversity is important to successful IT solutions

### PR DESCRIPTION
**Pull request by KaraWaldemer**

### CODE REVIEW

File: _posts/2020-07-11-diversity-for-the-right-solution.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+layout:      posts
+background:  shortBackground
```
Comment (Techneaux) [inline _posts/2020-07-11-diversity-for-the-right-solution.md:3]: Looks like the subtext gets cut off with shortBackground and mid screen resolutions. Suggest changing this to `mediumBackground`.
image

File: _posts/2020-07-11-diversity-for-the-right-solution.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+For many products, diversity can be the missing key. Diversity in all stages of the pipeline is critical in building not only the “best” solution but, more importantly, the “right” solution. Not sure you agree? Just look at the seat belt.
+
+![Seatbelt](/assets/images/posts/2020-07-10-diversity-for-the-right-solution/alexandria-gilliott-VBFbVw8kZdI-unsplash.jpg){:target="_blank"}
+
+Here is an example of how it is possible to do everything you can think of to build the best product and still get so much wrong. The seat belt has existed for a long time. It has been engineered and reengineered in attempts to improve it and cars go through rigorous testing dedicated to them and other safety features and yet it is still not “right” for many of its users. According to a [study](https://ajph.aphapublications.org/doi/10.2105/AJPH.2011.300275) published in 2011, women are 73% more likely than men to be injured in an accident when wearing a seatbelt.
```
Comment (Techneaux) [inline _posts/2020-07-11-diversity-for-the-right-solution.md:20]: Suggest adding `{:target="_blank"}` to the end of the study URL so it opens in a new tab

Comment (KaraWaldemer) [reply _posts/2020-07-11-diversity-for-the-right-solution.md:20]: Will do, thanks.

File: _posts/2020-07-11-diversity-for-the-right-solution.md (hunk 1 above)
Comment (KaraWaldemer) [reply _posts/2020-07-11-diversity-for-the-right-solution.md:3]: I started with mediumBackground but the `posted by` line gets hidden then...

Comment (Techneaux) [reply _posts/2020-07-11-diversity-for-the-right-solution.md:3]: Oh ugh, I just tried it and I see what you mean. Ok I revoke my suggestion! I can open a PR for fixing that spacing in general.

Comment (KaraWaldemer) [reply _posts/2020-07-11-diversity-for-the-right-solution.md:3]: i can add a new style that is in between the two in order to fix it for now and we can use your issue for a more permanent fix :)

**Comment by KaraWaldemer** [comment]
Changes made

**Pull request merged**